Sources/SwiftDocC/Semantics/ReferenceResolver.swift

```diff
@@ -363,8 +363,8 @@ struct ReferenceResolver: SemanticVisitor {
         switch node.name {
         case .conceptual(let documentTitle):
             return documentTitle
-        case .symbol(_):
-            return node.symbol?.names.title ?? ""
+        case .symbol(let declaration):
+            return node.symbol?.names.title ?? declaration.tokens.map { $0.description }.joined(separator: " ")
```

Comment:
I see ReferenceResolver.title used only in extension LinkDestinationSummary {..} only concern would be if there would be any case declaration.tokens would return nothing or description being empty. Do we need a guard to handle that scenario?

Reply:
The previous behavior was that an empty string was returned if the node didn't have a SymbolKit symbol so this doesn't change that, it just adds one more fallback.

If DocC created the node to represent a local symbol then it will have a SymbolKit symbol, so the fallback branch wouldn't get called.

Reply:
makes sense, thank you for explaining that.